src/sandbox.rkt

```diff
@@ -201,12 +201,17 @@
   (define target-alt-data
     (cond
       [(test-output test)
-       (define target-expr (test-output test))
-       (define target-train-errs (errors target-expr train-pcontext ctx))
-       (define target-test-errs (errors target-expr test-pcontext* ctx))
-       (alt-analysis (make-alt target-expr) target-train-errs target-test-errs)]
-      [else
-       #f]))
+        (define target-train-errs-list '())
+        (define target-test-errs-list '())
+
+        ;; When in platform, evaluate error
+        (for/list ([expr (in-list (test-output test))] #:when (cdr expr))
+          (let* ([target-expr (fpcore->prog (car expr) ctx)]
+                [target-train-errs (errors target-expr train-pcontext ctx)]
+                [target-test-errs (errors target-expr test-pcontext* ctx)])
```

Comment:
You can just use `(define ...)` in a `for` loop

Reply:
Altered!